docs/build/wasp-evm/v1.0.0-rc.6/docs/how-tos/integrate-your-exchange.mdx

```diff
@@ -0,0 +1,162 @@
+---
+tags: [Build, maintain, wasp, shimmerEVM, exchange ]
+description: Learn how to integrate ShimmerEVM to your exchange.  
+sidebar_label: Integrate Your Exchange
+---
+import { AddToMetaMaskButton, EVMNetworks } from '@theme/AddToMetaMaskButton';
+import { ChainId } from '@theme/ChainId';
+
+# ShimmerEVM Exchange Integration
+
+If you already support EVM, then you probably are already familiar with what you'll need to integrate ShimmerEVM. As
+with most networks, you should be able to use the same [JSON RPC API](https://wiki.iota.org/wasp-evm/reference/json-rpc-spec/) or clients, like [go-ethereum](https://geth.ethereum.org/), taking into account the
+[compatibility and limitations](../getting-started/compatibility.md), use the ShimmerEVM RPC
```

Comment:
this page linked is full of wrong/irrelevant info:
https://iota-wiki-9wsx0rnpl-iota1.vercel.app/wasp-evm/getting-started/compatibility/#properties-and-limitations

- why do we talk about "on-ledger" requests on "EVM limitations"? that's not part of the EVM...
- "Wrapped Calls to the JSON-RPC"  ??? why does this even matter? it's just an implementation detail.
- "Contract ID Source", "WASM Root Contract List", wtf...
- "EVM state resides in a raw form to prevent the duplication of efforts undertaken by the ISC layer" - what?

that entire page should be scrapped.


please just keep:
- `No Enforced Block Time`  - use the "shimmerEVM" avg block time instead
- `The Magic Contract`  - use "Layer 1 tokens" ,instead of ISC tokens , remove "generally avail"
- `Gas Fees`  - rewrite and simply state that "the gas price is set by the chain"